desk/app/expose.hoon

```diff
@@ -5,7 +5,7 @@
 ::    then visit in the browser:
 ::    /expose/that/reference/as/copied/123456789
 ::
-/-  c=cite, d=channels, co=contacts
+/-  c=cite, d=channels, t=contacts-0
```

Comment:
Can we keep the old import names, here and in other files, please?

Reply:
We can – although we avoid the name clash with `+co` arm in this way.